pkg/manager/impl/project_manager.go

```diff
@@ -60,6 +60,19 @@ func (m *ProjectManager) ListProjects(ctx context.Context, request admin.Project
 	}, nil
 }
 
+func (m *ProjectManager) UpdateProject(ctx context.Context, project admin.Project) (*admin.ProjectUpdateResponse, error) {
+	var response *admin.ProjectUpdateResponse
+
+	projectModel := transformers.CreateProjectModel(&project);
```

Comment:
are we assuming a fully formed project in the request and not a patch type operation?

Reply:
Not necessarily. We expect the ID and at least one of labels and description but there's no validation (yet). Domains are disregarded.